src/ol/disposable.js

```diff
@@ -0,0 +1,30 @@
+goog.provide('ol.Disposable');
+
+/**
+ * Objects that need to clean up after themselves.
+ * @constructor
+ */
+ol.Disposable = function() {};
```

Comment:
Actually, marking as `@struct` is creating compiler issues with `EventTarget` (where it uses bracket notation).  See https://travis-ci.org/openlayers/ol3/builds/113174215 for the failures.